### PR TITLE
Documents creator

### PR DESCRIPTION
**Pull request by MichaelOv**
Added Documents creator user guide

### CODE REVIEW

File: docs/user/basics/documents-creator.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Documents Creator
+description: Create user-friendly documents
+lang: en
+---
+
+# Documents Creator
+
+The Documents creator converts your documentation data into user-friendly documents. This allows you to reach a diverse audience - from tech-savvy stakeholders to the less digitally fluent decision-makers.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:9]: ...user-friendly PDF documents.

File: docs/user/basics/documents-creator.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+The documents can serve as a documentation basis for audits, conformity checks and customer reporting. Documents are available in the main navigation bar.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:11]: Original:
Documents are available in the main navigation bar.
Proposal:
The Documents Creator is available in the submenu of "Reports" in the main navigation bar

File: docs/user/basics/documents-creator.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Rights
+
+You should check the [Rights](../../admin/rights-and-permissions.md) and alter them to fit your needs. It is also possible to set `All available Rights (inlcuding future updates)`.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:15]: We have a variaty of rights know which can be described here:

* Download
* Manage documents
* Manage mailings

File: docs/user/basics/documents-creator.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Design your documents
+
+One size doesn't fit all, especially in the world of IT Asset Management. Different stakeholders have different needs. Customize your document with different chapters and sections and decide what elements of your IT asset data take center stage.
+
+### Global documents
+
+Global documents can be viewed by any tenant.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:23]: Global documents are available in all tenants and aim to fulfill the need to share documents. Whenever you want to populate a document with data from different tenants, global documents are the solution.

File: docs/user/basics/documents-creator.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+### Tenant documents
+
+Tenant documents are only available for the current tenant.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:27]: Tenant documents are designed to be only available for specific tenants. They cannot be shared or even viewed by unauthorized parties.

File: docs/user/basics/documents-creator.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Document settings
+
+Each document needs to have a Document title and can have a description.
+
+### Chapters
+
+There are default Chapter `Title page` and `Table of content`. Also there is a add button to add `custom chapter` for your needs.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:35]: `custom chapter` is not a buzzword or something special here. The user can just add new chapters as organizational unit to combine topics together. I would not use this word here.

File: docs/user/basics/documents-creator.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+#### Sections
+
+To fill the Chapter with data you have to Option to add `Object details` section, `Text` section, `Headline` section and a `Custom Table`. The `Headline` is added by default. In the `Text` section you can use supported [Markdown](https://www.markdownguide.org/basic-syntax/) to build high quality documents. Sections can be reordered or deleted.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:39]: to Option to add -> sounds weird.

Might be a good idea to make an unordered list from the section selection.

File: docs/user/basics/documents-creator.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+!!! tip "Text editor"
+    If you click on the `Text editor` button you can easily edit your text in the `Text editor` popup with a Toolbar and preview.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:42]: Text editor -> Markdown editor

File: docs/user/basics/documents-creator.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+In the `Object details` section the attributes are displayed per row.
+The `Custom Table` section displays the result like a table.´
+
+And you define conditions to narrow down the result set, by editing the `Objects shown in section`. Also you define the displayed `Attributes`.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:47]: And should not be at the beginning of the sentence...

File: docs/user/basics/documents-creator.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+### Sending documents to the target group
+
+Accelerate your communication: Instead of manually crafting and distributing documents, you can use automated mailings to swiftly and regularly compile the latest and most relevant information and send it to your audience.
+
+### Mailings
+
+Each mailing list needs a title, recipient(s) which can also be added by direct input, a subject and a message.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:55]: mailing list? sounds weird.

File: docs/user/basics/documents-creator.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+After that you can set a execution date and configure if the mailing should be done in an interval.
+When you check `Repeat execution` you can select a `End date of execution` and a interval.
```
Comment (skekec1) [inline docs/user/basics/documents-creator.md:57]: you can select a `End date of execution` and a interval
->
you can select `End date of execution` and an interval

File: docs/user/basics/documents-creator.md (hunk 8 above)
Comment (MichaelOv) [reply docs/user/basics/documents-creator.md:42]: Hm i wrote `Text editor` because that is the name of the Button and also of the modal

**Review by ljuergenliemke — APPROVED** [review]

**Pull request merged**